tests/raw_query/models.py

```diff
@@ -32,6 +32,10 @@ class Coffee(models.Model):
     price = models.DecimalField(max_digits=10, decimal_places=2, default=0)
 
 
+class Tea(models.Model):
+    id = models.AutoField(primary_key=True, db_column='TeA_iD')
```

Comment:
Are you sure that this actually creates a mixed-case column name? In the past, we needed to do things like `'"TeA_iD"'` to get this effect, otherwise the name would be uppercased or lowecased along the way.